mesh/mesh.hpp

```diff
@@ -832,7 +844,7 @@ class Mesh
        @return A bitmask:
        - bit 0 - simplices are present in the mesh (triangles, tets),
        - bit 1 - tensor product elements are present in the mesh (quads, hexes),
-       - bit 2 - the mesh has wedge elements.
+       - bit 2 - the mesh has wedge or pyramid elements.
```

Comment:
Pyramids should use bit 3.

Reply:
It certainly could but why should it? This function isn't used very often. Are you thinking people might use it to avoid using features that support wedges but not yet pyramids?

Reply:
The purpose of this is to be able to quickly tell what type of elements one has in the mesh, so we needs to distinguish wedges/prisms from pyramids -- that's all.

Reply:
Okay. That's not obvious from the function's description. Perhaps I'll expand that as well.

Reply:
> Perhaps I'll expand that as well.

Please do!